woocommerce/woocommerce-admin
# Fix tax report table search, add documentation.

Fixes #3365.

This PR adds documentation for the Taxes Report and fixes a bug with the report table search found while documenting.

### Screenshots

**Before:**

![Screen Shot 2020-03-03 at 11 37 46 AM](https://user-images.githubusercontent.com/63922/75832322-30d35580-5d73-11ea-89ad-8283b61a9958.png)

**After:**

![analytics-taxes-report-search-results](https://user-images.githubusercontent.com/63922/75832357-43e62580-5d73-11ea-8bad-28c44e1420cc.png)

### Detailed test instructions:

- Go to Analytics > Taxes
- Verify that the table partial string search works as expected
- Review documentation for typos and inaccuracies 



### Changelog Note:



Fix: Taxes Report search.
